ironfish/src/network/peers/peerManager.ts

```diff
@@ -144,13 +151,15 @@ export class PeerManager {
 
   constructor(
     localPeer: LocalPeer,
+    peerAddressManager: PeerAddressManager,
     logger: Logger = createRootLogger(),
     metrics?: MetricsMonitor,
     maxPeers = 10000,
     targetPeers = 50,
     logPeerMessages = false,
   ) {
     this.logger = logger.withTag('peermanager')
+    this.peerAddressManager = peerAddressManager
```

Comment:
nit: can we just call this addresses or something simple? or addressManager?

Reply:
👍🏽